maintainers/maintainer-list.nix

```diff
@@ -9157,6 +9157,12 @@
     githubId = 757752;
     name = "Jonas Heinrich";
   };
+  onthestairs = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.